### DISCUSSION TITLE
auth() was called but it looks like you aren't using `authMiddleware` in your middleware file

### DISCUSSION THREAD

**Discussion by frasermarch** [clerk/javascript#1236] [Support request - DO NOT OPEN HERE]
I'm using the nextjs app dir, and my middleware is definitely being called:

```
export default authMiddleware({
    afterAuth: async (auth, request, evt) => {
        console.log("hello from middleware")
    }
});


export const config = {
    matcher: ["/((?!.*\\..*|_next).*)", "/", "/(api|trpc)(.*)"],
};
```

But I still get the error that I'm not using authMiddleware when I use
```
   import { UserButton, auth } from '@clerk/nextjs';

    const user = auth()
```

Any suggestions?

**Comment by frasermarch** [comment]
I get the same issue when I use https://github.com/clerkinc/clerk-next-app-router-starter

**Reply by iamcrisb** [reply]
Where is your middleware file?

**Reply by frasermarch** [reply]
It was in the right place. Upgrading my version of node seemed to fix it

**Comment by frasermarch** [comment]
Solved by upgrading Node to v20
